CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+##  v1.10.0 (12/14/2020)
+
+**Common**
+
+* Added a shared_mutex class.
+* Added status logging of the network requests on iOS platform.
+* Fixed the proxy warming message on Android platform.
+* Added `Open` and `Close` API to `olp::cache::DefaultCache`. It can be used to open or close individual caches.
+* `olp::client::ApiLookupClient` now caches the `olp::client::OlpClient` instances internaly.
+* Added merge of the same requests in `olp::client::OlpClient` class. Internally it merges te same requests by URL and with empty payload.
+* Removed the unnecessary sleep in `olp::http::NetworkCurl` which forced all requests to be at least 100 ms.
+* Added downloaded size and time of execution for requests in `olp::http::NetworkCurl`.
+
+**olp-cpp-sdk-authentication**
+
+* Added a new GetMyAccount API. It can retrieve user informarmation for the issued access_token.
```

Comment:
```suggestion
* Added `GetMyAccount` API to `olp::authentication::AuthenticationClient`. It can retrieve user information based on a valid access token previously requested by the user upon sign-in.
```